index.js

```diff
@@ -1,7 +1,58 @@
 // Iteration 1: Names and Input
+let hacker1 = "Henrique";
+console.log("The driver's name is", hacker1);
 
+let hacker2 = "Guy";
+console.log("The navigator's name is", hacker2);
 
 // Iteration 2: Conditionals
+/*
+2.1. Depending on which name is longer, print:
+- The driver has the longest name, it has XX characters. or
+- It seems that the navigator has the longest name, it has XX characters. or
+- Wow, you both have equally long names, XX characters!.*/
+ 
+console.log(hacker1.length)
 
-
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters`);
+}
+else if (hacker1.length < hacker2.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+}
+else {
+    console.log(`Wow, you both have equally long names ${hacker1.length} characters!.`);
+}
 // Iteration 3: Loops
+/*3.1 Print the characters of the driver's name, separated by space, and in capital letters, i.e., "J O H N".*/
+
+let driver = ""
+for (i = 0; i < hacker1.length; i++) {
+    driver = driver + hacker1[i] + " ";
+}
+console.log(driver.toUpperCase());
```

Comment:
You should use the `.toUpperCase()` method inside the loop, otherwise you are not saving the string with the upper cased letters, you are just console logging it:

```
let driver = ""

for (i = 0; i < hacker1.length; i++) {
    driver = driver + hacker1[i].toUpperCase() + " ";
}

console.log(driver);
```